app/views/planning_applications/review/heads_of_terms/edit.html.erb

```diff
@@ -0,0 +1,50 @@
+<% content_for :page_title do %>
+  Review conditions - <%= t("page_title") %>
+<% end %>
+<%= render(
+      partial: "shared/review_task_breadcrumbs",
+      locals: {planning_application: @planning_application}
+    ) %>
+<% content_for :title, "Review heads of terms" %>
+<%= render(
+      partial: "shared/proposal_header",
+      locals: {heading: "Review heads of terms"}
+    ) %>
+
+<div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+  <div class="govuk-notification-banner__header">
+    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+      Important
+    </h2>
+  </div>
+  <div class="govuk-notification-banner__content">
+    <p class="govuk-notification-banner__heading">
+      This application requires a legal agreement.
+    </p>
+    <p>
+      Check whether heads of terms have been agreed.
+    </p>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h2 class="govuk-heading-m">Summary of heads of terms</h2>
+    <% if @planning_application.heads_of_term.present? %>
+      <ol class="govuk-list govuk-list--number">
+        <% @planning_application.heads_of_term&.terms.each do |term| %>
```

Comment:
If you are checking `@planning_application.heads_of_term.present?` not sure you need the safe navigator `&` here when checking `terms`